### ISSUE TITLE
Web Editor crashes on clicking Play/F5

### ISSUE THREAD

**Issue by nathanjwtx** [godotengine/godot#49894]
### Godot version

Web Editor v3.3.2

### System information

Windows 10

### Issue description

I am getting the errors displayed below on every project I have tried to run from the Web Editor.

I tried switching between GLES2 & 3 with the same outcome.

Edge v91.0.864.54

![image](https://user-images.githubusercontent.com/1558027/123318221-9a411f80-d4f4-11eb-80ca-00f73d95d6c6.png)

### Steps to reproduce

In theory, create a project in the version of Edge given above, then try to run it.

**Comment by nathanjwtx** [comment]
I get the exact same errors when trying under FireFox 89.0.2

**Comment by Calinou** [comment]
These errors are expected as the debugger protocol is not implemented in HTML5. The project should still run as expected (check the **Game** tab at the top).

**Comment by nathanjwtx** [comment]
Is this something that happens even if not debugging?

**Comment by Calinou** [comment]
> Is this something that happens even if not debugging?

Yes, the remote debugger is also used to display the project's prints to the Output tab.

**Comment by Faless** [comment]
You mention a "crash"... in the title.
Does it actually crash? Does the game not run?

**Comment by nathanjwtx** [comment]
no, i don't think it does actually; just slow to start. Guess I was being impatient :)

**Comment by lmsonic** [comment]
It runs for me on Windows 10 using Firefox 126.0.1 in Godot 4.2.2 (latest), using an empty project. Loads pretty fast also
